recipes/pyeit/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pyeit" %}
+{% set version = "1.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/liubenyuan/pyEIT/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6eac2f59a53386169bf677927b6177436e1909e916fd60ffe161f27c1a3680db
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy
+    - pandas
+    - python >=3.6
+    - scipy
+    - matplotlib-base
+    - shapely
+
+test:
+  imports:
+    - pyeit
+  source_files:
+    - tests/*.py
+  commands:
+    - pytest tests/
+  requires:
+    - pytest
+    - pytest-cov
+    - nose
```

Comment:
You only use `pytest` to run the tests.
`nose` isn't needed and you don't need `pytest-cov` either.